pkg/deployer/deployer.go

```diff
@@ -120,7 +120,7 @@ func (d *DeployerImpl) deploy(
 
 			publicIPCount := countDeploymentPublicIPs(dl)
 			log.Printf("Number of public ips: %d\n", publicIPCount)
-			contractID, err := sub.CreateNodeContract(d.identity, node, nil, hashHex, publicIPCount)
+			contractID, err := sub.CreateNodeContract(d.identity, node, "", hashHex, publicIPCount, nil)
```

Comment:
this shouldn't be always nil, should be passed from the schema

Reply:
https://github.com/threefoldtech/terraform-provider-grid/blob/development/examples/resources/peertube/main.tf#L29
should be added somewhere here